server/base/src/main/java/org/apache/accumulo/server/util/MasterMetadataUtil.java

```diff
@@ -149,7 +150,8 @@ private static KeyExtent fixSplit(ServerContext context, TableId tableId, Text m
 
           for (Entry<Key,Value> entry : scanner3) {
             if (entry.getKey().compareColumnFamily(DataFileColumnFamily.NAME) == 0) {
-              origDatafileSizes.put(new FileRef(context.getVolumeManager(), entry.getKey()),
+              String tabletFilePath = entry.getKey().getColumnQualifierData().toString();
+              origDatafileSizes.put(new FileRef(TabletFileUtil.validate(tabletFilePath)),
```

Comment:
This would be more readable if the validate call wrapped the right-hand-side of the assignment to `tabletFilePath` rather than being embedded into a constructor inside a parameter to the `put` method call.

Reply:
I 100% wrote it this way for nice formatting as any other way wraps the line of code.  I think your way make sense logically though.

Reply:
LOL, I wasn't thinking about formatting. Now, I'm not so sure. It might fit if `validate` were overloaded to take a `Text` so you didn't have to do `toString()`? Whatever you think best.

Reply:
Ah man... I don't like overloaded methods to begin with, let alone adding one purely for formatting haha.  I'll see how your suggestion fits and if its ugly ill just keep what I had.

Reply:
Either way, I think this could get replaced with ```TabletFile``` or ```Ample``` eventually.

Reply:
It's not purely formatting. I would argue that the overloaded method would help centralize the logic related to validation by simplifying calling code, and that, in turn, makes it easier to swap out later (because working with concrete types rather than Strings helps you create replacement code that satisfies all your caller use cases). Also, avoiding the `toString()` calls and using the concrete types instead, can help find a lot of situations where code is being unnecessarily converted back and forth to/from String.